flow/workflows/setup_flow.go

```diff
@@ -227,9 +224,14 @@ func (s *SetupFlowExecution) setupNormalizedTables(
 // executeSetupFlow executes the setup flow.
 func (s *SetupFlowExecution) executeSetupFlow(
 	ctx workflow.Context,
-	config *protos.FlowConnectionConfigs,
+	flowJobName string,
 ) (*protos.SetupFlowOutput, error) {
 	s.Info("executing setup flow")
+	// gotta fetch the config from the catalog.
+	config, err := internal.FetchConfigFromDB(flowJobName)
```

Comment:
can't call this in workflows

could maybe run as side effect, but then it ends up in history